src/ast.mli

```diff
@@ -1,4 +1,4 @@
-type program = None
+type program = message list * actor list * func list
```

Comment:
a program is a list of messages, actors, and functions. Where do strings come into play?